src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person: {
+  name,
+  age,
+  isMarried,
+  partnerName,
+} }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
+    {age && (
+      <p className="Person__age">{`I am ${age}`}</p>
+    )}
+    {isMarried ? (
+      <p className="Person__partner">{`${partnerName} is my husband`}</p>
+    ) : (
+      <p className="Person__partner">I am not married</p>
+    )}
```

Comment:
I think you can create a single `p` element and set the value depending on the value of isMarried